lib/customs.js

```diff
@@ -77,15 +77,15 @@ module.exports = function (log, error) {
     )
     .then(
       function (result) {
-        return { lockout: !!result.lockout }
+        return
       },
       function (err) {
         log.error({ op: 'customs.flag.1', email: email, err: err })
         // If this happens, either:
         // - (1) the url in config doesn't point to a real customs server
         // - (2) the customs server returned an internal server error
         // Either way, allow the request through so we fail open.
-        return { lockout: false }
+        return
```

Comment:
I did this because I went back and looked at the unit tests from before the account lockout patch was merged and the tests all explicitly checked for an undefined return value. I'm happy to update those tests to check for an empty object.